src/test/java/edu/unc/lib/boxc/migration/cdm/services/CdmIndexServiceTest.java

```diff
@@ -629,6 +646,48 @@ public void buildDocumentWithLongFieldTest() throws Exception {
         assertEquals("7", rootEl.getChildText("dmrecord"));
     }
 
+    @Test
+    public void indexFromCsvTest() throws Exception {
```

Comment:
we should probably add another test to verify that this works for a CSV file with more fields than the default ones from the export. I'd suggest using the fieldService.retrieveFieldsFromCsv method to get the fields from the provided CSV and then persisting them in this test and the new one as well, rather than using `cdm_export_fields.csv`